### ISSUE TITLE
First file in dir ignored

### ISSUE THREAD

**Issue by AlecRust** [svgstore/svgstore-cli#7]
svgstore looks like a great alternative to svg-sprite (which has a Phantom dep)!

However I've noticed that on the two projects I've switched to this, the first SVG in a directory is ignored and so missing from the end result sprite.

I'm using it like this:

```shell
svgstore -o public/images/icon-sprite.svg -p "i-" --inline public/images/svg-icons/*.svg
```

Whatever file is first (alphabetically) in `/svg-icons` is excluded from the `icon-sprite.svg` file. My current workaround is an empty `a.svg` file in the directory.

**Comment by shannonmoeller** [comment]
Well, that's no good. Bug confirmed. I'll take a look.

![image](https://user-images.githubusercontent.com/155164/32668653-6c756bb2-c5fb-11e7-86c4-742851ed807d.png)

**Comment by shannonmoeller** [comment]
Root cause is the `--inline` flag. The way yargs is setup, it thinks that the first matching file is the value for the `--inline` flag. Should just need to tell yargs that `--inline` is a boolean flag. On it.

**Comment by shannonmoeller** [comment]
Fixed. Released as v1.3.1.

**Comment by AlecRust** [comment]
Awesome, thank you!

**Comment by fritx** [comment]
Seems glob in svgstore-cli has bug.

I had to write:

```sh
svgstore -o assets/dist/sprite.svg assets/icons/**/*.svg assets/icons/*.svg --inline
```

instead of 

```sh
svgstore -o assets/dist/sprite.svg assets/icons/**/*.svg
```

---

Finally, I had to move all the icons to directories rather than leaving kind of them in the outer dir.

**Comment by BrianLenzo** [comment]
I encountered this issue where a directory with both files and more directories would only pick up the files. It wouldn't traverse the sub-directories and get any files matching the pattern in there. For me, this only happened on OSX/Linux and was fine on Windows.

After trying the fix above, we stumbled upon a better fix.  Add escaped quotes around your glob pattern in the command args and all files in any level directory would be picked up:

` "generate:glyphs": "svgstore -o dist/sprite.svg \"temp/glyph/**/*.svg\""`
